stdlib/public/runtime/EnvironmentVariables.cpp

```diff
@@ -187,9 +191,30 @@ extern "C" char **_environ;
 #endif
 #endif
 
+#if defined(__ANDROID__)
+// On Android, also try loading runtime debug env variables from system props.
+static void platformInitialize(void *context) {
+  (void)context;
+  char propValueString[PROP_VALUE_MAX] = "";
+#define SYSPROP_PREFIX "debug.org.swift.runtime."
```

Comment:
```suggestion
#define SYSPROP_PREFIX "org.swift.runtime.debug."
```

Reply:
Actually, do these variables persist in release mode? If so, perhaps we should drop the `debug.` component?

Reply:
Unfortunately, the `debug.` prefix is required. As I noted in the PR description:

> Only system properties prefixed with `debug.` can be set without root permissions.

I will add a comment to the code here to make it clear.